### DISCUSSION TITLE
Signal Strength between openDTU and converter

### DISCUSSION THREAD

**Discussion by huckes** [tbnobody/OpenDTU#579] [Q&A]
Hi, Is there any information available about the signal strength between the microconverter and the openDTU device? 

If yes, it could make sense to bring this information to the Web-interface, e.g. in Info - Network page or in the DTU settings as here is also the input field to set the PA level.

**Comment by njordan77** [comment]
yes, i would look for the same. any kind of information. is there anything sent in the packages or readable out of the logs?
